Documentation/Installation/Index.rst

```diff
@@ -0,0 +1,44 @@
+.. include:: ../Includes.txt
+
+
+
+.. _installation:
+
+============
+Installation
+============
+
+Target group: **Administrators**
+
+This extension is part of the TYPO3 core.
+
+Installation with composer
+==========================
+
+Check whether you are already using the extension with
+
+.. code-block:: bash
+
+   composer show | grep friendsoftypo3/dashboard
+
+This should either give you no result or something similar to:
+
+.. code-block:: bash
+
+   friendsoftypo3/dashboard                                    1.0.0            Dashboard for TYPO3
+
+If it is not yet installed, use the ``composer require`` command to install the extension
+
+.. code-block:: bash
+
+   composer require friendsoftypo3/dashboard ^1.0
```

Comment:
I would leave out the version here as this is about to change quite often I guess.